tests/foreman/ui/test_contentview.py

```diff
@@ -110,6 +113,53 @@ def test_no_blank_page_on_language_switch(session, target_sat, module_org):
         assert session.contentview_new.read_french_lang_cv()
 
 
+def test_republish_metadata(session, function_sca_manifest_org, target_sat):
+    """Verify that you can't republish metadata from the UI, and you can from the CLI
+
+    :id: 96ef4fe5-dec4-4919-aa4d-b8806d90b654
+
+    :steps:
+        1. Enable and Sync RH Repo
+        2. Add repo to a CV
+        3. Publish the CV
+        4. Navigate to the published Version's page
+        5. Verify that you can't click the "republish metadata" option from the UI
+        6. Verify that you can't republish metadata from the cli without the force option
+        7. Verify that you can republish metadata from the CLI using the --force option
+
+    :expectedresults: You can't republish RH Repo metadata from the UI, and can from the CLI with --force
+
+    :CaseImportance: Critical
+
+    :BZ: 2227271
+
+    :customerscenario: true
+    """
+    rh_repo_id = target_sat.api_factory.enable_sync_redhat_repo(
+        REPOS['rhae2.9_el8'], function_sca_manifest_org.id
+    )
+    rh_repo = target_sat.api.Repository(id=rh_repo_id).read()
+    cv = target_sat.api.ContentView(organization=function_sca_manifest_org).create()
+    cv = target_sat.api.ContentView(id=cv.id, repository=[rh_repo]).update(["repository"])
+    cv.publish()
+    version = cv.read().version[0].read()
+    with target_sat.ui_session() as session:
+        session.organization.select(org_name=function_sca_manifest_org.name)
+        assert session.contentview_new.republish_metadata_error(cv.name, 'Version 1.0')
+        with pytest.raises(CLIReturnCodeError) as error:
+            target_sat.cli.ContentView.version_republish_repositories(
+                {'id': version.id, 'force': 'false'}
+            )
```

Comment:
```suggestion
            target_sat.cli.ContentView.version_republish_repositories(
                {'id': version.id}
            )
```
Passing `'force': 'false'` is optional I guess, could you please check it locally and update here